src/test/java/org/apache/brooklyn/cloudfoundry/location/CloudFoundryLocationLiveTest.java

```diff
@@ -236,12 +207,52 @@ public void testCreateInstanceOfNonSupportedPlan() {
         cloudFoundryPaasLocation.createServiceInstance(params.getAllConfig());
     }
 
+    @Test(groups = {"Live"})
+    public void testBindServiceToApplication() {
+        ConfigBag params = getDefaultApplicationConfiguration();
+        createApplicationStartAndCheck(params.getAllConfig());
+        createServiceAndCheck(getDefaultClearDbServiceConfigMap());
+        assertFalse(cloudFoundryPaasLocation.isServiceBoundTo(SERVICE_INSTANCE_NAME, applicationName));
+        cloudFoundryPaasLocation.bindServiceToApplication(SERVICE_INSTANCE_NAME, applicationName);
```

Comment:
You can create an application.
Then, you can create an instance of a offered service by the platform marketplace, for example `clearDb` is a offered kind of service (which provides a MySql db).

Once, you have an application and a service instance, you can bind both of them.